src/Sylius/Component/Core/OrderProcessing/OrderShipmentProcessor.php

```diff
@@ -75,7 +76,7 @@ public function process(BaseOrderInterface $order)
     }
 
     /**
-     * @param BaseOrderInterface $order
+     * @param BaseOrderInterface | OrderInterface $order
```

Comment:
I added this because the method uses the `hasShipments` / `getShipments` / `addShipment` functionality of the order model, but those are not part of `Sylius\Component\Order\Model\OrderInterface`.

Can the definition of

```php
private function getOrderShipment(BaseOrderInterface $order)
```

be changed to

```php
private function getOrderShipment(OrderInterface $order)
```

so that the interface expectations align?